### DISCUSSION TITLE
Add a label / help / description to custom fields

### DISCUSSION THREAD

**Discussion by plogramer** [sadieblue/agilityblue.workspace-scripts#9] [Ideas]
When we add a custom field, it would be nice if we have a way to add some description or instruction for this new field.
Also, a way to group custom fields would be good, so we can visually put the related things together..

![image](https://github.com/user-attachments/assets/a5ed8ff0-f5c1-42ab-a08f-75f9b587460c)

**Comment by bmward80** [comment]
Hey @plogramer, thanks for the suggestion. This site is specifically related to scripting within AB, so you'll have better visibility into your suggestion if you post it on the general product feature request page located here: [https://ideas.agilityblue.com/boards/feature-requests](https://ideas.agilityblue.com/boards/feature-requests)

Brandon
